symless/jest-sonar-reporter
# Remove .npmignore and extra entries in files

https://symless.atlassian.net/browse/S3-1933

To test:

1. Install npm version:
    1. `npm i -g npm@latest`
    2. `npm i -g npm@9`
    3. `npm i -g npm@8`
    4. `npm i -g npm@7`
    5. `npm i -g npm@6`
2. Run `npm pack --dry-run`
3. Ensure that only the following files are present in the output:
    1. `LICENSE`
    2. `README.md`
    3. `package.json`
    4. `/lib/*`

**NOTE:** in `npm@6` pack / publish includes the CHANGELOG.md, however the `CHANGELOG*` entry in `.npmignore` does not prevent this.
